pkg/authorization/bootstrap/policy.go

```diff
@@ -46,6 +46,21 @@ const (
 	SystemKcpWorkspaceAccessGroup = "system:kcp:workspace:access"
 )
 
+const (
+	// SystemMastersGroup is the group inherited from k8s codebase - all powerful, all knowing!
+	// Users should not be added to this group.
+	SystemMastersGroup = "system:masters"
```

Comment:
this exists in Kube.

Reply:
You will be very surprised :D The only place where this constant in defined in kube is cmd/kubeadm :D everywhere else it just hardcoded :D I think nobody bothered :D 

If you give me hint where kube keeps constants like this, I can try cleaning :D

Reply:
https://github.com/kubernetes/kubernetes/blob/a499facee693a1a83daadb82d88f7b51d324ffc5/staging/src/k8s.io/apiserver/pkg/authentication/user/user.go#L71